source/_integrations/knx.markdown

```diff
@@ -1768,6 +1771,63 @@ entity_category:
   default: None
 {% endconfiguration %}
 
+## Time
+
+The KNX time platform allows to send time values to the KNX bus and update its state from received telegrams. It can optionally respond to read requests from the KNX bus with its current state.
+
+<div class='note'>
+
+Time entities without a `state_address` will restore their last known state after Home Assistant was restarted.
+
+Times having a `state_address` configured request their current state from the KNX bus.
+
+</div>
+
+<div class='note'>
+
+The `day` filed of the time telegram will always be set to 0 (`no day`).
+
+</div>
+
+```yaml
+# Example configuration.yaml entry
+knx:
+  time:
+    - name: "Time"
+      address: "0/0/2"
+      state_address: "0/0/2"
+```
+
+{% configuration %}
+name:
+  description: A name for this device used within Home Assistant.
+  required: false
+  type: string
+address:
+  description: Group address new values will be sent to. *DPT 10.001*
+  required: true
+  type: [string, list]
+state_address:
+  description: Group address for retrieving the state from the KNX bus. *DPT 10.001*
+  required: false
+  type: [string, list]
+respond_to_read:
+  description: Respond to GroupValueRead telegrams received to the configured `address`.
+  required: false
+  type: boolean
+  default: false
+sync_state:
+  description: Actively read the value from the bus. If `false` no GroupValueRead telegrams will be sent to the bus. `sync_state` can be set to `init` to just initialize state on startup, `expire <minutes>` to read the state from the KNX bus when no telegram was received for \<minutes\> or `every <minutes>` to update it regularly every \<minutes\>. Maximum value for \<minutes\> is 1440. If just a number is configured "expire"-behaviour is used. Defaults to `true` which is interpreted as "expire 60".
```

Comment:
Thank you very much for this suggestion! It reads way better.
Unfortunately it doesn't render as expected in `rake preview` 😬. I tried some variations and came up with a solution I think is ok - but this again triggers `markdownlint`.. I couldn't find a way to both render it properly and satisfy markdownlint. 
Do you have any idea how this would be possible?

Here is my current attempt (as in [ae549a6](https://github.com/home-assistant/home-assistant.io/pull/27979/commits/ae549a666091f8288e67ff80742d9b72b7ea0637)):
```md
sync_state:
  description: Actively read the value from the bus. The maximum time interval (`<minutes>`) is 1440. Following values are valid

    - `true` equivalent to "expire 60" (default)

    - `false` no GroupValueRead telegrams will be sent to the bus

    - `every <minutes>` to update it regularly every \<minutes\>

    - `expire <minutes>` to read the state from the KNX bus when no telegram was received for \<minutes\>

    - `<minutes>` equivalent to "expire \<minutes\>"

    - `init` to just initialize the state on startup

  required: false
  type: [boolean, string, integer]
  default: true
```

<img width="765" alt="Bildschirmfoto 2023-06-28 um 10 15 03" src="https://github.com/home-assistant/home-assistant.io/assets/12422879/17d62bf8-ba64-4d7f-b4f9-5b323ac375a3">

Reply:
Oh, nevermind... it seems that linter warning was just my local editor and it is not checked in CI 🙃